app/javascript/middleware/new-provider-reducer.ts

```diff
@@ -0,0 +1,10 @@
+import { AppState } from '../packs/miq-redux';
+
+const asdf = 'INIT_NEW_PROVIDER_HAWKULAR'
+function initNewProvider(state, action): AppState {
```

Comment:
I personally like a lot the spread operator, but in this case I was mostly referring to arrow functions vs functions (that require return etc)